src/metpy/calc/thermo.py

```diff
@@ -22,6 +22,13 @@
 exporter = Exporter(globals())
 
 sat_pressure_0c = units.Quantity(6.112, 'millibar')
+sat_pressure_0c_in_base_units = 611.2
+epsilon_in_base_units = mpconsts.epsilon.to_base_units().m
+Rd_in_base_units = mpconsts.Rd.to_base_units().m
+Lv_in_base_units = mpconsts.Lv.to_base_units().m
```

Comment:
Could we go with something like `base_unit_lv`? *mumbles something about stupid bots*